hack/test-integration.sh

```diff
@@ -13,14 +13,48 @@ cd "${OS_ROOT}"
 
```

Comment:
> why wouldn't we just set ETCD_PORT=44001 when we invoked the integration tests somewhere we wanted them parallelized?

There's no real reason to launch this script with default values other than 44001.  I'll make an envvar to override it, but no one will ever use it.